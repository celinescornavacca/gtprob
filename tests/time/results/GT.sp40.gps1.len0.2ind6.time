0.69user 0.16system 0:01.24elapsed 68%CPU (0avgtext+0avgdata 45132maxresident)k
136inputs+0outputs (0major+16627minor)pagefaults 0swaps
