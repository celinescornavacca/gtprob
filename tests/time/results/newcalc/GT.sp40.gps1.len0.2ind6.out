-42.9575101053
